generated/_task_modules/channels/ai_channel.py

```diff
@@ -20,7 +20,7 @@
     EddyCurrentProxProbeSensitivityUnits, ExcitationDCorAC,
```

Comment:
This is pretty close to the original codegen, but I don't think we should consider it done until there are no unexpected changes.

I flagged several sources of unexpected changes, but I didn't review the diff thoroughly because there were so many unexpected changes. Once you fix the ones I flagged, we may find others.